Stellar-operation-set-fees.x

```diff
@@ -15,58 +15,79 @@ namespace stellar
      */
     struct SetFeesOp
     {
+        //: Fee entry to set
         FeeEntry* fee;
-		bool isDelete;
-		// reserved for future use
-		union switch (LedgerVersion v)
-		{
-		case EMPTY_VERSION:
-			void;
-		}
-		ext;
+        //: `isDelete` indicates that fee should be either set or removed
+        bool isDelete;
+        //: reserved for future use
+        union switch (LedgerVersion v)
+        {
+        case EMPTY_VERSION:
+            void;
+        }
+        ext;
     };
 
     /******* SetFeesEmissionRequest Result ********/
-    
+
+    //: Result codes for SetFees operation
     enum SetFeesResultCode
     {
         // codes considered as "success" for the operation
+        //: `SetFeesOp` was successfully applied and fee was successfully set or deleted
         SUCCESS = 0,
-        
-        // codes considered as "failure" for the operation
-        INVALID_AMOUNT = -1,      // amount is negative
-		INVALID_FEE_TYPE = -2,     // operation type is invalid
+
+        // codes considered as "failure" for an operation
+        //: Fee amount is invalid (e.g. negative amount is ranked invalid)
+        INVALID_AMOUNT = -1,
+        //: `FeeType` which is set in `FeeEntry` struct is invalid (any `FeeType` not contained in `FeeType` enum is ranked invalid)
```

Comment:
//: `FeeType` which is set in the `FeeEntry` structure is invalid (any `FeeType` that is not contained in the `FeeType` enum is ranked invalid)